skin.tcl

```diff
@@ -225,6 +225,14 @@ proc iconik_show_settings {} {
 	show_settings $::settings(settings_profile_type)
 }
 
+proc iconik_select_profile {} {
+	show_settings
+	set_next_page off settings_1
```

Comment:
This seems to be the wrong order. Also it adds a new interaction that this can stop an espresso. Please remove the start_idle.
It also should be fine to just do a "show_settings settings_1"

Reply:
You are right. This was a careless mistake I made when just reusing one of the many different weird implementations of that functionality spread across the code base of different skins. However, `fill_profiles_listbox` was needed in addition. Otherwise the wrong profile is selected in listbox, after the current profile has been changed using one of the coffee buttons. I updated the code, thanks in advance for checking.